types/dec_coin.go

```diff
@@ -551,7 +551,7 @@ func ParseDecCoins(coinsStr string) (coins DecCoins, err error) {
 		return nil, nil
 	}
 
-	splitRe := regexp.MustCompile(",|;")
+	splitRe := regexp.MustCompile(",")
 	coinStrs := splitRe.Split(coinsStr, -1)
```

Comment:
There is only one delimiting, `regexp` can be replaced by `strings.Split`

Reply:
Indeed.

Reply:
Updated.